SocialCareCaseViewerApi.Tests/V1/Helpers/TestHelpers.cs

```diff
@@ -96,11 +96,39 @@ public static CaseNoteResponse CreateCaseNoteResponse(CaseNote? caseNote)
                 .RuleFor(c => c.AllocatedTeamId, f => teamId ?? team.Id)
                 .RuleFor(c => c.AllocatedWorkerId, f => workerId ?? worker.Id)
                 .RuleFor(c => c.CreatedBy, f => createdBy ?? createdByWorker.Email)
+                .RuleFor(c => c.RagRating, f => f.Random.String2(5))
```

Comment:
if column sizes are set in the schema file, then they could be reflected in the length of these strings